index.html

```diff
@@ -142,58 +142,53 @@ <h2>About <br> Myself</h2>
           <li><a href="#"><img src="images/M.jpg" alt="M"></a></li>
         </ul>
         <nav>
-          <button type="button" class="button">Get my resume</button>
+          <button type="button" class="button-2">Get my resume</button>
         </nav>
       </div>
+      </div>  
+      
+      <section class="others">
+          <article class="my-works">
+            <h3>Languages</h3><i class="fa-solid fa-angle-down"></i>
+          </article>
+            <ul class="lang">
+              <li><img src="images/javascript logo.jpg" alt="javascript"><p>Javascript</p></li>
+              <li><img src="images/HTML logo.jpg" alt="HTML"><p>HTML</p></li>
+              <li><img src="images/css logo.jpg" alt="css"><p>CSS</p></li>
+            </ul>
+          <article class="my-works">
+            <h3>Frameworks</h3><i class="fa-solid fa-angle-right"></i>
+          </article>
+          <article class="my-works">
+            <h3>Skills</h3><i class="fa-solid fa-angle-right"></i>
+          </article>
+      </section>
+    </section>
+  </main>
 
-      <div class="others">
-        <ul>
-          <li>Languages<a href="#"><span><img src="images/Union.jpg"></a></span>
-            <div id="lang">
-              <a href="#"><img src="images/Langguage List (1).jpg" width="270"></a>
-            </div>
-          </li>
-          <li>Frameworks<a href="#"><span><img src="images/Union 2.jpg"></span></a>
-            <div>
-              <a href="#"><img src="images/Separator Bottom.jpg" width="270"></a>
-            </div>
-          </li>
-          <li>Skills<a href="#"><span id="two"><img src="images/Union 2.jpg"></span></a>
-            <div>
-              <a href="#"><img src="images/Separator Bottom.jpg" width="270"></a>
-            </div>
-          </li>
-        </ul>
-      </div>
+  <section class="formsection">
+    <div class="heading-2">
+      <h2>Contact me</h2>
```

Comment:
-  You are doing great, Two main points to make your design align with the Figma design. 
- [ ] 1. The elements of the form should be centered on the page including the Get in touch button.
- [ ] 2. we need to apply the appropriate backgrounds.

## Your design
![your-form](https://user-images.githubusercontent.com/69077061/201469519-2345b33b-7389-4baa-bd32-e0a221446deb.PNG)

## Figma design
![figma-form](https://user-images.githubusercontent.com/69077061/201469533-2be2905c-d79a-47fa-8a32-62e6e7beeb4b.PNG)

Reply:
it is centered. i used animation so it aligns in the middle in 2s.